src/processes/helper/preTopicAutoMsg.js

```diff
@@ -0,0 +1,35 @@
+const momentTz = require('moment-timezone');
+const {sample} = require('lodash');
+const {automateWelcomeMsgQueue} = require('../../config');
+const {LogError} = require('../../databases/models');
+
+const preTopicAutoMsgProcess = async (returnPrepopulatedDm) => {
+  try {
+    let currentTime = momentTz().tz('America/Los_Angeles');
+
+    const randomTime = sample([6, 7, 8]);
+    const additionalDays = 1;
+    let requiredTime = momentTz()
+      .tz('America/Los_Angeles')
+      .set({hour: randomTime})
+      .add(additionalDays, 'days');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for calculating the `requiredTime` is not clear. If you want to add randomness, consider adding comments to explain why you are doing this and how it works.

17-18, 22:
Avoid using `console.log` statements in production code. If you need to log these values, consider using a proper logging library that can be configured to output logs based on the environment (development, production, etc.).

24:
The line of code that adds a job to the queue is commented out. If this is intentional and you plan to enable it later, consider adding a comment explaining why it's currently disabled.



<!-- This is an auto-generated comment by OSS CodeRabbit -->